Formula/e/enzyme.rb

```diff
@@ -2,10 +2,9 @@ class Enzyme < Formula
   desc "High-performance automatic differentiation of LLVM"
   homepage "https://enzyme.mit.edu"
   # TODO: Check if we can use unversioned `llvm` at version bump.
```

Comment:
Let's try this.

Reply:
Ah okay, I only noticed https://github.com/EnzymeAD/Enzyme/commit/116e8bbc5289f793ea6aa0972fd2212d5ca46401. Let's do this later then. Thanks for checking!

Reply:
👍 💯